app/Http/Requests/StoreUpdateRegistrationRequest.php

```diff
@@ -36,6 +36,10 @@ public function rules()
             'carers.*' => 'distinct|string|different:carer',
             // MAY be present; MUST be a date format of '2017-07'
             'children.*' => 'date_format:Y-m',
+            // MAY be null (if not present) or 0||1
+            'fm_chart' => 'nullable|in:0,1',
```

Comment:
would TRUE or FALSE be more clear?

Reply:
I'm using 0,1 for safety.

Laravel doesn't cast "true"/"false" very well, as (bool) "false" returns true. The string "false" has content so it's true.